### DISCUSSION TITLE
Livewire compatibility?

### DISCUSSION THREAD

**Discussion by binaryfire** [spatie/laravel-responsecache#458] [Q&A]
Hi all. Just wondering if it’s possible to use this package with Livewire 3?

**Comment by emaia** [comment]
Hi! It's possible. I'm using it on a project right now... I don't know exactly why but to make it work You need to manually load livewire [https://livewire.laravel.com/docs/installation#manually-including-livewires-frontend-assets](url)

**Reply by binaryfire** [reply]
@emaia Thanks, will give that a shot

**Reply by HassanZahirnia** [reply]
Hey @emaia have you actually managed to cache Livewire's post update requests? If so can you please share your cache profile ?

I tried to do it but apparently it keeps returning the same cache for all requests.

**Comment by SadeghPM** [comment]
i think there must be a replacer like csrf. is it?

**Reply by iRajul** [reply]
can someone explain how to make it work with livewire csrf post request?
